dune-project

```diff
@@ -33,6 +33,8 @@
   (ocaml
    (>= 4.14.0))
   dune
+  (ocamlformat
```

Comment:
ocamlformat isn't a dependency of the project so we shouldn't add it here

Reply:
Actually, I put it here to trigger this discussion :-)

If it's not installed, automatic PR checks are likely to fail as they are using it and the dev is not. Although not technically a dependency of the project, it is nevertheless on the critical path for seamless maintenance. 

An alternative could be to have `make fmt` in a git pre-push hook. But one could argue that would make ocamlformat a dependency :-)

Reply:
Having it installed alongside the other dependencies won't help to enforce that contributors format their code. A git hook is an option, but I think it requires manual steps to setup? So we'd need to document it and it's not a given that contributors will do it. So it doesn't completely solve the problem either.

IMO asking contributors to format their code when we see that the CI check failed is low friction and I don't see a solution that removes the need for a CI check (or reduces the odds that the check will fail)